.eslintrc.yml

```diff
@@ -1,11 +1,17 @@
-extends: eslint:recommended
+extends:
+  - eslint:recommended
 
 env:
-  node: true
-  es6: true
   browser: true
+  es6: true
+  node: true
+
+root: true
 
 rules:
   comma-dangle: off
+  indent: [error, 2]
   no-console: off
   no-unused-vars: [error, { vars: all, args: none }]
+  quotes: [error, double]
+  semi: [error, never]
```

Comment:
Added a couple new rules, notably `indent`, `quotes`, and `semi` since we were inconsistent.